TESTING.md

```diff
@@ -1,89 +1,483 @@
+          "params": {
+            "resources": {
+              "path": "/.vib/$$ASSET$$/ginkgo"
+            },
+            "params": {
+              "kubeconfig": "{{kubeconfig}}",
+              "namespace": "{{namespace}}",
+              "$$PARAM_3$$": "$$VALUE_3$$",
+              ...
+            }
+          }
+        }
+```
+
+### Run it locally
+
+Sometimes it is of interest to run the tests locally, for example during development. Though there may be different approaches, you may follow the steps below to execute the tests locally:
+
+1. Deploy the target Chart in your cluster, using the same installation parameters specified in the `vib-verify.json` pipeline file
+
+    ```bash
+    $ cat .vib/metallb/vib-verify.json | grep "runtime_parameters"
+        "runtime_parameters": "Y29udHJvbGxlcjoKICBwb2RTZWN1cml0eUNvbnRleHQ6CiAgICBlbmFibGVkOiB0cnVlCiAgICBmc0dyb3VwOiAxMDAyCiAgY29udGFpbmVyU2VjdXJpdHlDb250ZXh0OgogICAgZW5hYmxlZDogdHJ1ZQogICAgcnVuQXNVc2VyOiAxMDAyCiAgICByZWFkT25seVJvb3RGaWxlc3lzdGVtOiBmYWxzZQogICAgY2FwYWJpbGl0aWVzOgogICAgICBkcm9wOgogICAgICAgIC0gQUxMCiAgc2VydmljZUFjY291bnQ6CiAgICBjcmVhdGU6IHRydWUKICAgIGF1dG9tb3VudFNlcnZpY2VBY2NvdW50VG9rZW46IHRydWU=",
+    $ helm install metallb bitnami/metallb -f <(echo "Y29udHJvbGxlcjoKICBwb2RTZWN1cml0eUNvbnRleHQ6CiAgICBlbmFibGVkOiB0cnVlCiAgICBmc0dyb3VwOiAxMDAyCiAgY29udGFpbmVyU2VjdXJpdHlDb250ZXh0OgogICAgZW5hYmxlZDogdHJ1ZQogICAgcnVuQXNVc2VyOiAxMDAyCiAgICByZWFkT25seVJvb3RGaWxlc3lzdGVtOiBmYWxzZQogICAgY2FwYWJpbGl0aWVzOgogICAgICBkcm9wOgogICAgICAgIC0gQUxMCiAgc2VydmljZUFjY291bnQ6CiAgICBjcmVhdGU6IHRydWUKICAgIGF1dG9tb3VudFNlcnZpY2VBY2NvdW50VG9rZW46IHRydWU=" | base64 -d)
+    ```
+
+1. Download and [install Ginkgo](https://onsi.github.io/ginkgo/#installing-ginkgo) in your system
+1. Execute the tests. Provide the necessary params (usually, the path to the kubeconfig file and namespace name, but check `vib-verify.yaml`).
+
+    ```bash
+    $ cd .vib/metallb/ginkgo
+    $ ginkgo -- --kubeconfig=./kube.config --namespace=default
+      Running Suite: MetalLB Integration Tests - /.vib/metallb/ginkgo
+      ===============================================================
+      Random Seed: 1674578785
+      Will run 1 of 1 specs
+
+      Ran 1 of 1 Specs in 1.341 seconds
+      SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 0 Skipped
+      PASS
+
+      Ginkgo ran 1 suite in 6.895174927s
+      Test Suite Passed
+    ```
+
+### Specific acceptance criteria
+
+* [ ] Test file name has the following format: Helm chart name + `test` (ex: `metallb_test.go`)
+* [ ] Helper functions should be placed in an additional file named `integration_suite_test.go`
+* [ ] Created cluster resources must be cleaned up after execution
+
+## GOSS
+
+[GOSS](https://github.com/aelsabbahy/goss/blob/master/docs/manual.md) is one of the frameworks used to implement integration tests. Related files should be located under `/.vib/ASSET/goss`. It is the reference tool to use when tests require interaction with a specific pod.
```

Comment:
I would also add that, differently from Cypress and Ginkgo, Goss is executed from inside the pod.